Project EFT/Data Classes/StandardUser.cs

```diff
@@ -43,5 +43,45 @@ public void UpdateRanking()
             int newRanking = DBConnector.GetUserRanking(Id);
             if (newRanking != DBConnector.DB_FAILURE) Ranking = newRanking;
         }
+
+        public bool IsEqualWithSubMap(StandardUser otherUser)
+        {
+            bool subMapEqual = true;
+            bool shouldBreak = false;
+
+            //checks the equality of the values in the submission dictionary....
+            foreach (KeyValuePair<int, List<AnswerSubmission>> kv in otherUser.Submissions)
+            {
+
+                if (this.Submissions.ContainsKey(kv.Key))
+                {
+                    int count = 0;
+                    foreach (AnswerSubmission a in kv.Value)
+                    {
+                        if (!this.Submissions[kv.Key][count].IsEqual(a))
+                        {
+                            subMapEqual = false;
+                            shouldBreak = true;
+                        }
+                        count++;
```

Comment:
sub map not equal --> users not equal, so it could just return false inside of the conditional on 61, and if the conditional never evals to true, it's implied that the submissions are equal

Reply:
Yep, thats what my logic was doing but more convoluted.   I just wasnt thinking about the easier way to do that.  Updated.